modalsheet/src/main/java/eu/wewox/modalsheet/ModalSheet.kt

```diff
@@ -55,11 +55,15 @@ import kotlin.math.roundToInt
  * @param scrim Optional custom scrim.
  * @param content The content of the bottom sheet.
  */
+@ExperimentalSheetApi
 @Composable
-fun ModalSheet(
+public fun ModalSheet(
```

Comment:
Doesn't `public` redundant here? Even IDE warns about it 🤔 

I thought `public` is a default visibility 🤷 Or it is different for libraries?

Reply:
Did you do a Gradle sync? That's the Explicit API mode. **Everything** has to have a visibility modifier

Reply:
Damn, sry, did not understand correctly how it should work 👍